src/services/integration/index.ts

```diff
@@ -199,4 +200,15 @@ export class IntegrationManager extends BaseAdapterManager implements DefaultInt
       avatarConfig,
     });
   };
+
+  /**
+   * @function onRoomInfoUpdate
+   * @description room update
+   * @param {} room
+   * @returns {void}
+   */
+  private onRoomInfoUpdate = (room): void => {
+    const properties = room._customProperties;
+    this.adapter.setFollow(properties.followUserId);
+  };
 }
```

Comment:
Can you add type to the room? This is the room type 

https://github.com/SuperViz/sdk/blob/113bd1732e5809f7ea949a41020e837c8cc3a4e2/src/services/realtime/ably/types.ts#L14

```suggestion
  /**
   * @function onRoomInfoUpdate
   * @description room update
   * @param {AblyRealtimeData} room
   * @returns {void}
   */
  private onRoomInfoUpdate = (room: AblyRealtimeData): void => {
    const properties = room._customProperties;
    this.adapter.setFollow(properties.followUserId);
  };
}
```